pa1-3

```diff
@@ -0,0 +1,25 @@
+# COMP 3410
+# Author: Will Robb
+# Assignment: PA1 Part 3
+# Date: 02/11/15
+
+.text
+main:
+	li $s0, 1		# odd integer register
+	li $s1, 0		# sum register
+	li $t0, 0		# counter register
+
+loop: 
+	bge $t0, 8, endloop	# end loop after 8 passes
+	add $s1, $s0, $s1	# add previous sum to next odd int
+	addi $s0, $s0, 2	# increment by two for next odd int
+	addi $t0, $t0, 1	# increment counter register
+	j loop			# jump to start of the loop
```

Comment:
-6 Instructions require that you store partial sum in memory and retrieve it for operation on each iteration.